src/view/pay-lightning-confirm-mobile.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import MainContent from '../component/main-content';
+import { NamedField } from '../component/field';
+import { Header, Title } from '../component/header';
+import { CancelButton, BackButton, GlasButton } from '../component/button';
+import Card from '../component/card';
+import LightningBoltIcon from '../asset/icon/lightning-bolt';
+import { FormStretcher } from '../component/form';
+import {
+  BalanceLabel,
+  BalanceLabelNumeral,
+  BalanceLabelUnit,
+} from '../component/label';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  balance: {
+    marginBottom: 10,
+  },
+  numeral: {
+    color: color.blackText,
+  },
+  unit: {
+    color: color.blackText,
+  },
+  totalLbl: {
+    marginTop: 5,
+  },
+  note: {
+    marginTop: 5,
+    borderBottomWidth: 0,
+  },
+  confirmBtn: {
+    marginTop: 20,
+  },
+});
+
+const PayLightningConfirmView = ({ store, nav, payment }) => (
+  <Background color={color.purple}>
+    <Header color={color.purple}>
+      <BackButton onPress={() => nav.goPay()} />
+      <Title title="Lightning Payment">
+        <LightningBoltIcon height={12} width={6.1} />
+      </Title>
+      <CancelButton onPress={() => nav.goHome()} />
+    </Header>
+    <MainContent>
```

Comment:
Seems to be missing the copy: https://projects.invisionapp.com/share/P2NS9FNN7VD#/screens/299894025 "You are about to send a bitcoin payment over the lightning network"

Reply:
Yeah, we removed this later when we added the invoice description since the screen felt a bit crowded. It was also kind of redundant to the Title "Lightning Payment".